addon.py

```diff
@@ -91,6 +99,48 @@ def show_conference(conf):
         url = plugin.url_for(resolve_event,
                              event=event['url'].rsplit('/', 1)[1])
         addDirectoryItem(plugin.handle, url, item, False)
+
+    if relive is not None:
+        relive_item = ListItem('ReLive (unreleased)')
+        url = plugin.url_for(show_relive, conf=conf)
+        addDirectoryItem(plugin.handle, url, relive_item, True)
+
+    endOfDirectory(plugin.handle)
+
+
+@plugin.route('/relive/<conf>')
+def show_relive(conf):
+    data = None
+    try:
+        relive = list(filter(lambda x: x.project == conf, http.fetch_relive().recordings))
+        relive = relive[0] if len(relive) == 1 else None
+        url = urlparse(relive.index_url, 'https').geturl()
```

Comment:
I used urlparse here instead of taking the url directly from the json because it didn't have a scheme in the jsons I looked at. E.g. `//cdn.c3voc.de/relive/rc3/203581/muxed.mp4`

Reply:
Yeah, those URLs are protocol-relative, so maybe resolve against the URL of the JSON you took them from?

Reply:
It's also always https when resolving against the URL I took it from as the index url is harcoded https which has protocol-relative links to the recordings listing (so these should be https) which in turn has relative links to the individual recordings, so these would also be https.
I would thus spare the effort and leave it hardcoded here